lib/plugins/leetcode.js

```diff
@@ -594,14 +601,36 @@ plugin.githubLogin = function(user, cb) {
       if (resp.statusCode !== 200) {
         return cb('GitHub login failed');
       }
-      _request.get({url: leetcodeUrl}, function(e, resp, body) {
-        const redirectUri = resp.request.uri.href;
-        if (redirectUri !== 'https://leetcode.com/') {
-          return cb('GitHub login failed or GitHub did not link to LeetCode');
+      if (resp.request.uri.href == 'https://github.com/sessions/two-factor') {
```

Comment:
Those request strings can also be extracted as configurations. In case that some day the URL changes, we can easily find it and update it.